release_notes/ocp-4-8-release-notes.adoc

```diff
@@ -1335,6 +1334,7 @@ As part of the continued deprecation of the Operator package manifest format, th
 
 *Insights Operator*
 
+* Previously, the Insights Operator did not collect Cluster Version Operator (CVO) pods or events in the `openshift-cluster-version` namespace. As a result, the Insights Operator did not display information about any problems that the CVO might experience. As a result, users could not get diagnostic information about the CVO. The Insights Operator was changed to gather the CVO pods and events from the `openshift-cluster-operator` namespace. As a result, issues with the the CVO are reorted by the Insights Operator. (link:https://bugzilla.redhat.com/show_bug.cgi?id=1942271[*BZ#1942271*])
```

Comment:
```suggestion
* Previously, the Insights Operator did not collect Cluster Version Operator (CVO) pods or events in the `openshift-cluster-version` namespace. As a result, the Insights Operator did not display information about any problems that the CVO might experience, and users could not get diagnostic information about the CVO. The Insights Operator is now updated to gather the CVO pods and events from the `openshift-cluster-operator` namespace so that issues with the the CVO are reported by the Insights Operator. (link:https://bugzilla.redhat.com/show_bug.cgi?id=1942271[*BZ#1942271*])
```
Just trying to avoid so many `As a result`s